WordPress/src/main/java/org/wordpress/android/ui/sitecreation/SiteCreationMainVM.kt

```diff
@@ -112,6 +114,7 @@ class SiteCreationMainVM @Inject constructor(
         if (isStarted) return
         if (savedInstanceState == null) {
             tracker.trackSiteCreationAccessed(siteCreationSource)
+            tracker.trackSiteCreationDomainPurchasingExperimentVariation(domainPurchasingExperiment.getVariation())
```

Comment:
Sounds good to me @antonis 🙇 

I was thinking as well that it might be even better to make sure the event is only fired if the feature flag is enabled 👍 

I'll update the code ;)

Reply:
@antonis I applied your suggestion to track the experiment event only when the feature is enabled within this commit: https://github.com/wordpress-mobile/WordPress-Android/pull/17945/commits/ebdcd3f8ecf62369f911d62aa7eac1cb52793b2d

After iterating a bit on this, I thought about it and figured out it would be nice if we can actually control via the feature config (debug settings) when the event is tracked 👍 

Let me know wdyt of this proposal 😉

Reply:
Thank you for the changes @ovitrif 🙇 
The new approach LGTM. I've also retested the flow and everything works as expected 🎉